templates/Yves/Plugin/Router/RouteProviderPlugin.php.twig

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace {{ namespace }};
+
+use Spryker\Yves\Router\Plugin\RouteProvider\AbstractRouteProviderPlugin;
+use Spryker\Yves\Router\Route\RouteCollection;
+
+class {{ classname }} extends AbstractRouteProviderPlugin
+{
+    protected const {{ bundle | upper }}_INDEX = '{{ bundleDashed | lower }}-index';
```

Comment:
We actually don't need this constant. We can generate url itself inside of the method

Reply:
@gechetspr  Updated plugin template, remove this constant. Pls check